### ISSUE TITLE
[Adaptive Cards> Input.text]: Ensures buttons have discernible text.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#8451]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25314.1010)
Browser: Edge Dev
URL: https://adaptivecards.io/explorer/Input.Text.html
Tool: Accessibility Insights for web

**Repro steps:**
1. Open the above URL in edge dev browser.
2. Run the fast pass of accessibility Insights for web.
3. Observe the issue.

**Actual Result:**
Buttons does not have discernible text.

**Expected Result:**
Ensures Buttons have discernible text.

**Snippet**: <button class="ac-inlineActionButton iconOnly ac-selectable" tabindex="0" role="button" style="margin-left: 8px;"><img role="presentation" src="https://adaptivecards.io/content/send.png" style="height: 100%;"></button>

**How to fix:** 
Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

**User Impact:**
Screen reader user will get impacted as user will not be able to know the purpose of the button and will not be able to navigate or interact with the content easily.

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/s/accessibility/EZBde1C58eFKkdEvHy1sfB0BbfrE-sgqsVO5g2zVY6wbDA?e=jEWXd4

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

<img width="960" alt="Code snippet~ Ensures buttons discernible text" src="https://user-images.githubusercontent.com/104483005/230545964-3c0a0fda-f884-412f-a4b4-0d92028e0a76.png">
<img width="960" alt="MAS4 1 2 Ensures buttons discernible text" src="https://user-images.githubusercontent.com/104483005/230546001-39643157-f2e6-454f-b8cc-772250c9aafc.png">

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Hi @anna-dingler,

As this issue is still repro, so could you please let us know is there any future plan to fix this issue?

Snippet:

![image](https://github.com/microsoft/AdaptiveCards/assets/95913748/8976b67d-c7b7-4947-80ad-b2f739625cf8)

**Comment by anna-dingler** [comment]
Hi @vagpt,

This issue is fixed on our internal site. I expect us to make another website release this month, so I am marking this as completed.

Please let me know if you have any questions!

**Comment by vagpt** [comment]
Hi @anna-dingler,

Thanks for the updates.

We have one query here i.e. 

As per your comment can we assume complete Adaptive cards website UI is getting changed or just the fixes will be merged on the current website?

**Comment by vagpt** [comment]
Hi @anna-dingler,

Please provide us confirmation on above ask. Also, could you please confirm on which environment do we need to perform A11y Test pass?

Thanks,

**Comment by anna-dingler** [comment]
Hi @vagpt,

Apologies for the late response. I just released a new version of our website, and the fix can be verified at https://adaptivecards.io/

-Anna
